python/ray/actor.py

```diff
@@ -715,6 +716,21 @@ def _remote(self, args=None, kwargs=None, **actor_options):
         placement_group_capture_child_tasks = actor_options[
             "placement_group_capture_child_tasks"
         ]
+        if placement_group != "default":
+            warnings.warn(
+                "DeprecationWarning: placement_group parameter is "
+                "deprecated. Use scheduling_strategy parameter instead."
+            )
+        if placement_group_bundle_index != -1:
+            warnings.warn(
+                "DeprecationWarning: placement_group_bundle_index is "
+                "deprecated. Use scheduling_strategy parameter instead."
+            )
+        if placement_group_capture_child_tasks:
```

Comment:
Can you add a test this is not warned when the new APIs are used? (PlacementGroupScheudlingStrategy)

Reply:
Done.